.github/workflows/create-release.yml

```diff
@@ -0,0 +1,81 @@
+name: Create github release & release notes
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+      - master
+    types: [ labeled ]
```

Comment:
How do we plan on running this workflow? I think the current configuration will potentially clash with the current PR beta publish workflow.

Maybe we could tie this down to PRs to main only? Then it won't clash with the other one that is only to develop.